connect/common/src/main/buf.gen.yaml

```diff
@@ -0,0 +1,21 @@
+#
```

Comment:
I suggest the directory name of `spark-connect` to capture that connect is only for spark.

Reply:
@tdas Hmm, from an outside look, even though the content of the `python` folder is only Spark, there is nothing in the naming that suggests that it is for Spark no? Shall we also do `delta/spark-connect`?

Reply:
the entire content of python/delta is for spark. I agree that ideally this entire python/delta should be under spark/, but thats an existing problem that we will solve holistically. 

its not clear to me yet, why all the python files need to be under python/delta/connect/ and not directly under python/delta/

can you tell me more?

Reply:
I dont have a strong opinion. for now we can keep it here... and we can move it when we have more clarity.

Reply:
@tdas Because all the files under `python/delta/connect` are for Delta **Connect**, perhaps we can change the name to `delta-connect` also.

I think naming `name1/name2` might be better than `name1-name2`, so `delta/connect` instead of `delta-connect`, `spark/connect` instead of `spark-connect`.